### ISSUE TITLE
Sometimes its give error while extracting

### ISSUE THREAD

**Issue by ankit29kothari** [SergeyPirogov/webdriver_manager#103]
```
Looking for [IEDriverServer 3.150.1 x64] driver in cache 
Trying to download new driver from http://selenium-release.storage.googleapis.com/3.150/IEDriverServer_x64_3.150.1.zip
Unpack archive C:\Users\mcjp2518\.wdm\drivers\IEDriverServer\3.150.1\x64\IEDriverServer.zip
Traceback (most recent call last):
  File "C:\Users\mcjp2518\Desktop\step2successs\Step2success\Selenium-Web Automation\easy_selenium\window.Authenication.Test.py", line 3, in <module>
    driver=open_browser(browser='ie',path=r'C:\Users\mcjp2518\Desktop\step2successs')
  File "C:\Users\mcjp2518\Desktop\step2successs\Step2success\Selenium-Web Automation\easy_selenium\easyselenium\easyselenium.py", line 75, in open_browser
    driver = webdriver.Ie(IEDriverManager().install())
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\microsoft.py", line 21, in install
    return self.download_driver(self.driver)
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\manager.py", line 39, in download_driver
    path = self.__download_and_save_driver_to_cache(driver, driver_version)
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\manager.py", line 29, in __download_and_save_driver_to_cache
    driver.get_os_type())
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\driver_cache.py", line 64, in save_driver_to_cache
    files = self.__unpack(file_path)
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\driver_cache.py", line 113, in __unpack
    return extract_zip(path, to_directory)
  File "C:\Program Files\Python37\lib\site-packages\webdriver_manager\archive.py", line 7, in extract_zip
    archive.extractall(to_directory)
  File "C:\Program Files\Python37\lib\zipfile.py", line 1616, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Program Files\Python37\lib\zipfile.py", line 1670, in _extract_member
    open(targetpath, "wb") as target:
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\mcjp2518\\.wdm\\drivers\\IEDriverServer\\3.150.1\\x64\\IEDriverServer.exe'
```

**Comment by jaromrax** [comment]
Hi, 
   I got completely stuck:

from python3 I launch
```
from selenium import webdriver
from webdriver_manager.chrome import ChromeDriverManager
driver = webdriver.Chrome(ChromeDriverManager().install())
```
and, despite it worked earlier, I get
```
[WDM] - Trying to download new driver from http://chromedriver.storage.googleapis.com/83.0.4103.39/chromedriver_linux64.zip
[WDM] - Unpack archive /home/***/.wdm/drivers/chromedriver/83.0.4103.39/linux64/chromedriver.zip
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/***/.local/lib/python3.6/site-packages/webdriver_manager/chrome.py", line 28, in install
...
...
  File "/usr/lib/python3.6/zipfile.py", line 1578, in _extract_member
    open(targetpath, "wb") as target:
OSError: [Errno 26] Text file busy: '/home/***/.wdm/drivers/chromedriver/83.0.4103.39/linux64/chromedriver'
```

Only when I remove `rm`
/home/***/.wdm/drivers/chromedriver/83.0.4103.39/linux64/chromedriver

I can start over. How to avoid the necessity to rm file?

Thanks

**Comment by SergeyPirogov** [comment]
How frequently it's happening. Do you run your tests in parallel?

**Comment by SergeyPirogov** [comment]
@jaromrax

**Comment by jacarrasco** [comment]
I can replicate the issue. It is happening when Cache is expiring. 

Last working log:
06/04/2020 18:37:11 [INFO] logger: GH_TOKEN will be used to perform requests
06/04/2020 18:37:11 [INFO] logger: Cache is valid for [04/06/2020]
06/04/2020 18:37:11 [INFO] logger: Looking for [geckodriver v0.26.0 linux64] driver in cache 
06/04/2020 18:37:11 [INFO] logger: Driver found in cache [/home/ubuntu/.wdm/drivers/geckodriver/v0.26.0/linux64/geckodriver]

When Cache is over:
06/05/2020 01:00:13 [INFO] logger: GH_TOKEN will be used to perform requests
06/05/2020 01:00:13 [INFO] logger: Getting latest mozilla release info for v0.26.0
06/05/2020 01:00:13 [INFO] logger: Trying to download new driver from https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-linux64.tar.gz
06/05/2020 01:00:14 [INFO] logger: Unpack archive /home/ubuntu/.wdm/drivers/geckodriver/v0.26.0/linux64/geckodriver-v0.26.0-linux64.tar.gz
06/05/2020 01:00:14 [INFO] extractor: Exception: [Errno 26] Text file busy: '/home/ubuntu/.wdm/drivers/geckodriver/v0.26.0/linux64/geckodriver'
06/05/2020 01:00:14 [INFO] extractor: <class 'OSError'>
06/05/2020 01:00:14 [INFO] extractor: [Errno 26] Text file busy: '/home/ubuntu/.wdm/drivers/geckodriver/v0.26.0/linux64/geckodriver'

**Comment by SergeyPirogov** [comment]
That's because you have the driver process running. Try to use ps aux to look for any active driver processes. For now, I don't know is the correct way to handle this. 
Maybe you have ideas?

**Comment by SergeyPirogov** [comment]
Fixed by #128
